docs/reference/docs/bulk.asciidoc

```diff
@@ -197,6 +197,16 @@ size for your particular workload.
 If using the HTTP API, make sure that the client does not send HTTP
 chunks, as this will slow things down.
 
+[float]
+[[bulk-optimistic-concurrency-control]]
+=== Optimistic Concurrency Control
+
+Each index and delete bulk item can include the `if_seq_no` and `if_primary_term`
+parameters in their respective action and meta data lines. These parameters
```

Comment:
Each `index` and `delete` action within a bulk API call may include the `if_seq_no` and `if_primary_term` parameters in their respective action and meta data lines.

Reply:
thx. adpate.d